communities.json

```
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "modulz.svg",
+      "title": "Modulz",
+      "quote": "We use Discord to offer support, gather feedback and announce product releases. But above all, to connect and engage with our community.",
+      "inviteCode": "p8PwAPFNHc",
```

Want to update this invite code to your new vanity url?
```suggestion
      "inviteCode": "modulz",
```

---

hell yeah! looks great. Thanks!